core/src/scss/utilities/mixins/shadow/_box-shadow.scss

```diff
@@ -11,6 +11,9 @@
 // Style guide: Mixins.Shadow.Box Shadow
 //
 @mixin box-shadow($depth: 'flat', $stroke: '1px solid #C6C6C6') {
-  border: #{$stroke};
+  @if $stroke != 0 and $stroke != 'none' and $stroke != null {
```

Comment:
Why the extra validation?

Reply:
> Why the extra validation?

Oh, that's because sometimes I just want the shadow and no border, and if I use the mixin it will write `border: 0`. Sometimes that overrides other custom borders styles I have on those elements on other sites, so I'd rather it doesn't output a border property at all when there's no border.